src/ripple/json/tests/Object.test.cpp

```diff
@@ -221,8 +215,8 @@ class JsonObject_test : public TestOutputSuite
 #ifdef DEBUG
         setup ("repeating keys");
```

Comment:
I'm confused why this is wrapped in an ifdef DEBUG. Is there a reason I'm missing?